src/components/Popups/PopupHighlight.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import noop from 'lodash/noop';
+import { FormattedMessage } from 'react-intl';
+import IconHighlightTextAnnotation from 'box-ui-elements/es/icons/annotations/IconHighlightTextAnnotation';
+import messages from './messages';
+import PopupBase from './PopupBase';
+import { DOMRectMini } from '../../store';
```

Comment:
I don't think we want regular components importing anything from the store. We also seem to have a duplicate type, Shape, that represents the same object. Can we clean up the code for converting to/from rect/bounding rect?

Reply:
Sure, I will use the new `Shape` type created in scroll PR